stubs/docker-compose.yml

```diff
@@ -43,6 +44,19 @@ services:
             - sail
         healthcheck:
           test: ["CMD", "mysqladmin", "ping"]
+#    pgsql:
+#        image: postgres:12
```

Comment:
Why 12 instead of 13?

Reply:
Wasn't on purpose. I think we can safely change it to v13, but just to be 100% sure I can test it before the push.

Reply:
I've just tested it, everything looks working